### PR TITLE
Clean up ATI, Chartbeat & Scripts

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Clean up ATI, Chartbeat & Scripts

**Code changes:**

- Moves reused code into cypress commands and uses in frontpages and articles tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/integration/pages/frontPage/amp.js [hunk 1]
```diff
@@ -21,6 +22,17 @@ Object.keys(config)
...
+      describe('ATI', () => {
+        it('should have an amp-analytics tag with the ati url', () => {
+          cy.hasAmpAnalyticsAtiUrl(
+            envConfig.atiUrl,
+            config[service].isWorldService
+              ? envConfig.atiAnalyticsWSBucket
+              : '',
+          );
+        });
+      });
```
Comment (sareh) [inline cypress/integration/pages/frontPage/amp.js:34]: For completeness, we should have a test that checks for scripts on AMP pages too, e.g. A cypress command `cy.hasExpectedAmpScripts`.

Comment (sareh) [reply cypress/integration/pages/frontPage/amp.js:34]: Likewise, front pages of both platforms should have a Chartbeat cypress test that asserts to not have the Chartbeat script.

File: cypress/integration/pages/article/canonical.js [hunk 2]
```diff
@@ -176,6 +176,36 @@ Object.keys(config)
...
+          cy.hasNoscriptImgAtiUrl(
+            envConfig.atiUrl,
+            config[service].isWorldService
+              ? envConfig.atiAnalyticsWSBucket
+              : '',
+          );
+        });
+      });
+
+      describe('Chartbeat', () => {
```
Comment (12) [inline cypress/integration/pages/article/canonical.js:190]: Chartbeat is toggled off at the moment on live. Are we going to expect Cypress to fail when things are toggled off?

File: cypress/integration/pages/frontPage/amp.js (hunk 1 above)
Comment (dr3) [reply cypress/integration/pages/frontPage/amp.js:34]: Agreed, not in scope for this PR though, its just moving stuff

File: cypress/integration/pages/article/canonical.js (hunk 2 above)
Comment (dr3) [reply cypress/integration/pages/article/canonical.js:190]: Gah, Fixed

File: cypress/integration/pages/frontPage/amp.js (hunk 1 above)
Comment (jamesbhobbs) [reply cypress/integration/pages/frontPage/amp.js:34]: @dr3 can you open an issue for this, also referencing the TODO we added last night pls

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Just add an issue to cover the amp bundles pls

File: cypress/integration/pages/frontPage/amp.js (hunk 1 above)
Comment (dr3) [reply cypress/integration/pages/frontPage/amp.js:34]: https://github.com/bbc/simorgh/issues/2948 :)

**Review by jamesbhobbs — APPROVED** [review]

**Review by 12 — APPROVED** [review]

**Pull request merged**